modules/olm-mirroring-package-manifest-catalog.adoc

```diff
@@ -40,18 +40,19 @@ On your workstation with unrestricted network access, run the following command:
 [source,terminal]
 ----
 $ oc adm catalog mirror \
-    <registry_host_name>:<port>/olm/redhat-operators:v1 \//<1>
-    <registry_host_name>:<port> \
-    [-a ${REG_CREDS}] \//<2>
-    [--insecure] \//<3>
-    [--index-filter-by-os='<platform>/<arch>'] \//<4>
-    [--manifests-only] <5>
-----
-<1> Specify your Operator catalog image.
-<2> Optional: If required, specify the location of your registry credentials file.
-<3> Optional: If you do not want to configure trust for the target registry, add the `--insecure` flag.
-<4> Optional: Specify which platform and architecture of the catalog image to select when multiple variants are available. Images are passed as `'<platform>/<arch>[/<variant>]'`. This does not apply to images referenced by the catalog image. Valid values are `linux/amd64`, `linux/ppc64le`, and `linux/s390x`.
-<5> Optional: Only generate the manifests required for mirroring and do not actually mirror the image content to a registry.
+    <registry_host_name>:<port>/olm/redhat-operators:v1 \ <.>
+    <registry_host_name>:<port> \ <.>
+    [-a ${REG_CREDS}] \ <.>
+    [--insecure] \ <.>
+    [--index-filter-by-os='<platform>/<arch>'] \ <.>
+    [--manifests-only] <.>
+----
+<.> Specify your Operator catalog image.
+<.> Specify the fully qualified domain name (FQDN) for the target registry.
```

Comment:
This is the only actually-new line. The rest of the surrounding changes are just switching to `.` for automated callout numbering instead of incrementing each number.